UI/ObjectListWnd.cpp

```diff
@@ -2409,9 +2409,15 @@ void ObjectListWnd::ObjectRightClicked(GG::ListBox::iterator it, const GG::Pt& p
         return;
 
     const int MENUITEM_SET_FOCUS_BASE = 20;
+    const int MENUITEM_SET_SHIP_BASE = 50;
+    const int MENUITEM_SET_BUILDING_BASE = 100;
```

Comment:
need to space out these numbers a lot more for user-customizable stuff... a player could easily have more than 50 producible ship designs. alternatively, add some limits so never more than the 50 most commonly-used ship designs are listed in the droplist.